styles.css

```diff
@@ -759,3 +759,9 @@ Media query for desktop version at 768px
 form {
   max-width: 447px;
 }
+@media(max-width:767px){
+  .desktop-nav {
+    display: none;
```

Comment:
#6  Kindly consider the name of the class desktop nav where it is being used in mobile media query to make your code easier to read.